site-src/references/implementations.md

```diff
@@ -7,18 +7,20 @@ Implementors of Gateway API are encouraged to update this document with status i
 ## Implementation Status
 
 - [Contour][1] (alpha)
-- [Gloo Edge 2.0][2] (work in progress)
-- [Google Kubernetes Engine][3] (work in progress)
-- [Istio][4] (alpha)
-- [Kong][5] (work in progress)
-- [Traefik][6] (alpha)
+- [Emissary-Ingress (Ambassador API Gateway)][2] (alpha)
+- [Gloo Edge 2.0][3] (work in progress)
+- [Google Kubernetes Engine][4] (work in progress)
+- [Istio][5] (alpha)
+- [Kong][6] (work in progress)
+- [Traefik][7] (alpha)
 
 [1]:#contour
-[2]:#gloo-edge
-[3]:#google-cloud-platform
-[4]:#istio
-[5]:#kong
-[6]:#traefik
+[2]:#emissary
```

Comment:
Unfortunately this link doesn't work - it looks like the following change would work:

```suggestion
[2]:#emissary-ingress-ambassador-api-gateway
```